### PR TITLE
Lesson1 6

### PR DESCRIPTION
**Pull request by SergeDot**

### CODE REVIEW

**Comment by SergeDot** [comment]
Will resolve the conflicts once the previous PR is merged

File: src/InputWithLabel.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { useEffect, useRef } from 'react';
+
+const InputWithLabel = ({ todoTitle, handleTitleChange, children }) => {
+  const inputRef = useRef();
+  useEffect(() => {
+    inputRef.current.focus();
+  });
```
Comment (riousenkai) [inline src/InputWithLabel.jsx:7]: I know that instructions are telling you not to have a dependency array. For small applications like this, it's okay to do this. For the most part you should always add a dependency array in your useEffect. That way the useEffect does not trigger on every re-render.

No change needed, just wanted to point that out.

Comment (SergeDot) [reply src/InputWithLabel.jsx:7]: Got it thank you!

**Comment by riousenkai** [comment]
Just to be safe - let's not merge stuff before I approve. Thanks.

**Pull request merged**